phpBB/viewforum.php

```diff
@@ -710,10 +710,28 @@
 		'SELECT'		=> $sql_array['SELECT'],
 		'FROM'			=> $sql_array['FROM'],
 		'LEFT_JOIN'		=> $sql_array['LEFT_JOIN'],
-
 		'WHERE'			=> $db->sql_in_set('t.topic_id', $topic_list),
 	);
 
+	/**
+	* Event to modify the SQL query before obtaining topics/stickies
```

Comment:
I would prefer multiline comments to have the asterix indented with a space:
```
/*
 * A comment
 */
```
rather than
```
/*
* A comment
*/
```

Reply:
Same here ,
in this event : https://github.com/dark-1/phpbb/blob/ticket/16381/phpBB/phpbb/session.php#L1090
there it's used & it's `@since 3.2.9-RC1` ???

Reply:
Yes and there are many more places.
Unfortunately there is no consistency in those yet, hence I remarked it. 
We should move towards a more uniform way. 
Not sure how others feel about this, but I feel this is the way to go.

These comments apply to all your event PR's.

Reply:
For this particular case,
If I go by this : https://area51.phpbb.com/docs/32x/coding-guidelines.html
& this : https://area51.phpbb.com/docs/33x/coding-guidelines.html
I have to use :
```php
/**
* A comment
*/
```
So I think it's better to keep as-per the `coding-guidelines` ...

Reply:
And that's what I have been trying to discuss for quite some time, see http://area51.phpbb.com/phpBB/viewtopic.php?f=81&t=70561 aswell
Not indenting the asterix with a space is very old code style.
All proper editors automatically indent these block comments.
Have a look through the vendor folder, and see how many (if not all) use it.

Those coding guidelines need to be updated.
And the code has to be cleaned up throughout the core aswell, some is just several years old.

Reply:
So should I wait till coding guidelines is updated ?

Reply:
I think I'll keep it as per `coding-guidelines` , once your discussion topic comes on a final decision, and the `coding-guidelines` are updated , I'll follow that henceforth.